app/src/entry-point.lisp

```diff
@@ -299,7 +305,15 @@ Copyright (c) 2016-2019 Rigetti Computing.~2%")
 
   (when (plusp qubit-limit)
     (setf *qubit-limit* qubit-limit))
-  
+
+  (unless (null parallelization-limit)
+    (cond
+      ((<= 1 parallelization-limit 50)
+       (setf qvm::*qubits-required-for-parallelization* parallelization-limit))
+      (t
+       (warn "Wacky parallelization limit of `~D' provided; ignoring it."
```

Comment:
Necessary? If someone wants to be wacky maybe let them

Reply:
A zero or negative number doesn't make sense. We could error in this case, what do you think?

Reply:
Sure, ignore it for `<= 0`. Don't prevent the wacky scientists from using `> 50` tho

Reply:
It's actually a type-level thing that's required.